code/game/mecha/mecha_construction_paths.dm

```diff
@@ -1591,3 +1591,216 @@
 		..()
 		feedback_inc("mecha_odysseus_created",1)
 		return
+
+
+/////// Hermes
+
+/datum/construction/mecha/hermes_chassis
+	steps = list(list("key"=/obj/item/mecha_parts/part/hermes_torso),//1
+					 list("key"=/obj/item/mecha_parts/part/hermes_head),//2
+					 list("key"=/obj/item/mecha_parts/part/hermes_left_arm),//3
+					 list("key"=/obj/item/mecha_parts/part/hermes_right_arm),//4
+					 list("key"=/obj/item/mecha_parts/part/hermes_left_leg),//5
+					 list("key"=/obj/item/mecha_parts/part/hermes_right_leg)//6
+					)

```

Comment:
Is there a need for such bizarre indentation?

Reply:
It's an attempt to align with tabs, as opposed to spaces.